packages/runtime-core/src/component.ts

```
@@ -98,6 +98,13 @@ import type { RendererElement } from './renderer'
 
 export type Data = Record<string, unknown>
 
+/**
+ * For extending allowed non-declared attrs on components in TSX
+ */
+export interface AllowedAttrs {}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)